### PR TITLE
Analysis Update: Add EWS subtyping workflow to README

### PR DESCRIPTION
**Pull request by zzgeng**
#### What scientific question is your analysis addressing?
Add EWS subtyping workflow to README

#### What was your approach?
Add EWS subtyping workflow to README

#### Which areas should receive a particularly close look?
EWS subtyping

#### Reproducibility Checklist

- [ ] The dependencies required to run the code in this pull request have been added to the project Dockerfile.
- [ ] This analysis has been added to continuous integration.

#### Documentation Checklist

- [ ] This analysis module has a `README` and it is up to date.
- [ ] This analysis is recorded in the table in `analyses/README.md` and the entry is up to date.
- [ ] The analytical code is documented and contains comments.

### CODE REVIEW

File: analyses/molecular-subtyping-EWS/README.md [hunk 1]
```diff
@@ -13,6 +13,11 @@ This module can be run with the following:
...
 ```
 
+## Molecular subtyping EWS workflow
+
+ ![EWS molecular subtyping workflow](./plot/EWS_subtyping.png)
+ 
+
 ### Module contents
 
 `01-run-subtyping-ewings.Rmd` identifies sample_id for RNA-Seq samples with hallmark _EWSR1_ fusions and matches sample_id with WGS sample and adds molecular_subtype as "EWS" (Ewing Sarcoma)
```
Comment (AntoniaChroni) [inline analyses/molecular-subtyping-EWS/README.md]:
```suggestion
`01-run-subtyping-ewings.Rmd` identifies sample_id for RNA-Seq samples with hallmark _EWSR1_ fusions and matches sample_id with WGS sample and adds molecular_subtype as "EWS" (Ewing Sarcoma).
```

**Review by AntoniaChroni — APPROVED** [review]
The EWS classification seems to be coming from identification of `_EWSR1_ fusions` from the RNA seq data. In the figure, it is marked DNA methylation instead.

**Review by naqvia — APPROVED** [review]
Code looks good, but the flow diagram should say fusion as that is what defines EWS.

**Review by jharenza — CHANGES_REQUESTED** [review]
Hi @zzgeng - in the subtyping (maybe this is unclear in the readme) - we are checking path dx == EWS OR methylation classification == EWS, so can you update that here? If the path dx is not EWS, then samples undergo pathology review before we classify as EWS, and that should also be captured.

**Review by jharenza — APPROVED** [review]

**Comment by jharenza** [comment]
docs only, no need for GA

**Pull request merged**